src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -309,8 +310,24 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                         ->cannotBeEmpty()
                                     ->end()
                                     ->scalarNode('initial_place')
+                                        ->beforeNormalization()
+                                            ->ifTrue(function ($v) { return null !== $v;})
+                                            ->then(function ($v) {
+                                                @trigger_error('The "initial_place" configuration key has been deprecated in Symfony 4.3. Use the "initial_places" configuration key instead.', E_USER_DEPRECATED);
+
+                                                return $v;
+                                            })
+                                        ->end()
                                         ->defaultNull()
                                     ->end()
+                                    ->arrayNode('initial_places')
+                                        ->beforeNormalization()
+                                            ->ifTrue(function ($v) { return !\is_array($v); })
```

Comment:
should we not display an error when marking store is `single_state` ? to allow only one initial place

Reply:
Good catch. I will do that in the extension though